src/main/java/uk/gov/hmcts/reform/idam/web/strategic/SPIService.java

```diff
@@ -329,7 +329,7 @@ protected boolean isRedirectable(String method) {
         HttpEntity<Void> entity = new HttpEntity<>(headers);
 
         return restTemplate.exchange(configurationProperties.getStrategic().getService().getUrl() + "/" + configurationProperties.getStrategic().getEndpoint().getValidateResetPasswordToken(), HttpMethod.POST, entity,
-            String.class);
+            Object.class);
```

Comment:
doesn't seem you're using the response from this method, yet this was changed.
is anything else using this response or could it be void?

Reply:
not sure this was resolved?